news/4910.bugfix

```diff
@@ -0,0 +1,9 @@
+Workaround for a Windows issue (encountered for example on Windows 10 machines,
+even with no anti-virus software installed) where a folder removal operation may
+fail while it actually does remove the folder or at least parts of it and a
+repeated operation then completes successfully. This used to cause pip to report
+failures on repeated operations due to the target folder not existing. Now such
+use-cases are treated as successful folder removals.
+
+One externally facing change is that the ``rmtree()`` operation now no longer
```

Comment:
this doesn't matter since pip doesn't actually have an API as such.

Reply:
@pradyunsg - Would it be better to just extract that last paragraph into a separate `.misc` changelog entry then?

Reply:
I think it can just be dropped entirely.

Reply:
Ok, dropped that last sentence.